dotcom-rendering/src/components/ArticleTitle.stories.tsx

```diff
@@ -1,616 +1,246 @@
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-ArticleSeriesTag.storyName = 'Article with a Series tag';
-ArticleSeriesTag.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: Pillar.Sport,
-			design: ArticleDesign.Standard,
-		},
-	]),
-];
-
-export const SpecialReportTitle = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: ArticleSpecial.SpecialReport,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Special',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-SpecialReportTitle.storyName = 'Special report';
-SpecialReportTitle.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: ArticleSpecial.SpecialReport,
-			design: ArticleDesign.Standard,
-		},
-	]),
-];
-
-export const SpecialReportAlt = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: ArticleSpecial.SpecialReportAlt,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Special Report Alt',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-SpecialReportAlt.storyName = 'Special report Alt';
-SpecialReportAlt.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: ArticleSpecial.SpecialReportAlt,
-			design: ArticleDesign.Standard,
+	},
+	parameters: {
+		colourSchemeBackground: {
+			light: '#005689',
+			dark: '#004e7c',
 		},
-	]),
-];
-
-export const ArticleNoTags = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: Pillar.Culture,
-					design: ArticleDesign.Standard,
-				}}
-			/>
-		</Wrapper>
-	);
-};
-ArticleNoTags.storyName = 'Article with no tags';
-ArticleNoTags.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: Pillar.Culture,
-			design: ArticleDesign.Standard,
+		colourSchemeTextColour: {
+			light: '#fff',
+			dark: '#fff',
 		},
-	]),
-];
+	},
+	name: 'LiveBlog Design, with blog tag',
+} satisfies Story;
 
-export const LabsStory = () => {
-	return (
-		<Wrapper>
-			<ArticleTitle
-				{...FEArticle}
-				format={{
-					display: ArticleDisplay.Standard,
-					theme: ArticleSpecial.Labs,
-					design: ArticleDesign.Standard,
-				}}
-				tags={[
-					{
-						id: '',
-						title: 'Series title',
-						type: 'Series',
-					},
-				]}
-			/>
-		</Wrapper>
-	);
-};
-LabsStory.storyName = 'Labs';
-LabsStory.decorators = [
-	splitTheme([
-		{
-			display: ArticleDisplay.Standard,
-			theme: ArticleSpecial.Labs,
-			design: ArticleDesign.Standard,
+export const LiveBlogDesignWithBlogTagAndMatch = {
+	args: {
+		...LiveBlogDesignWithBlogTag.args,
+		isMatch: true,
+	},
+	parameters: {
+		colourSchemeBackground: {
+			light: '#ffe500',
+			dark: '#f3c100',
```

Comment:
Same question as above.

![image](https://github.com/user-attachments/assets/53e5ed67-f34d-485a-a8f3-4a37fbd1aca2)

Reply:
These were set in the previous stories:

https://github.com/guardian/dotcom-rendering/blob/ffce9fda6710ea915d59b42ac16a09574aa36d29/dotcom-rendering/src/components/ArticleTitle.stories.tsx#L265-L267

and match what we have set in the palette:

https://github.com/guardian/dotcom-rendering/blob/ffce9fda6710ea915d59b42ac16a09574aa36d29/dotcom-rendering/src/palette.ts#L3632-L3635

I think there's a way we can use the palette directly in stories for this, but it will require a larger change to the theme decorators.